### PR TITLE
[GITHUB-2] Allows agent to be stopped by default

### PR DESCRIPTION
**Pull request by lobsterdore**
Added autostart option that to allow the agent to be started at
a later date. Also ensured that the service is stopped and disabled
so it won't start on machine boot up.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -7,5 +7,6 @@
 
 - name: Configure datadog Agent
   include: configure.yml
+  when: datadog.enabled
```
Comment (wojtek-oledzki) [inline tasks/main.yml:10]: :+1:

**Comment by wojtek-oledzki** [comment]
Pure Awesomeness!

Pure Awesomeness

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Develop: [![Build Status](https://travis-ci.org/sansible/datadog.svg?branch=deve
 
 This roles installs the Datadog agent.
 
+The agent service is intentionally disabled on bootup; during boot a machine will most likely
+consume a high amount of resources whilst the OS and all of it's services start which will trip any
+CPU or Memory usage alarmserroneously.
```
Comment (sreddel) [inline README.md]: more spaces required?

**Comment by sreddel** [comment]
good solution :+1:

**Pull request merged**